packages/design-system/src/components/snackbar/snackbarContainer.js

```diff
@@ -151,13 +165,13 @@ export const SnackbarContainer = ({
               <ChildContainer placement={placement}>
                 <Component
                   {...notificationProps}
-                  aria-label={message}
+                  aria-hidden
```

Comment:
Don't need this to be visible to screen readers now that the message is being announced. The global shortcut will also remove the need to hit the action button in the snackbar with the keyboard.

Note: it was already really tricky (maybe not possible?) to tab into the snackbar

Reply:
yea these feel a little weird to tab into